nailgun/entities.py

```diff
@@ -1487,7 +1506,7 @@ def __init__(self, server_config=None, **kwargs):
                 OperatingSystem,
                 null=True,
             ),
-            'parent': entity_fields.OneToOneField(HostGroup, null=True),
+            'ancestry': entity_fields.StringField(null=True),
```

Comment:
Does this field reference a `HostGroup` entity? If so, I'm very wary about changing this from a `OneToOneField` to a `StringField`.